silverstripe/silverstripe-admin
# FIX Support nested redux forms

## Description

Allows nested redux-forms to be submitted without propogating the nested submit event up to the parent form.

## Manual testing steps
1. Add the associated PRs (see issue) to your site
1. Upload a file to your site
1. Set up an elemental block that has links (single or multi doesn't matter)
2. Add a file link
3. Select "choose existing" and then select the file
4. Make sure your browser window width is less than 992px wide (the "insert" button when you've selected the file should become "insert file")
5. Click "insert file" (this should add the file to the upload field and **NOT** submit the link form)
6. Save the link (this should not give you a console warning about not returning a promise)

## Issues

- https://github.com/silverstripe/silverstripe-admin/issues/1662

## Pull request checklist

    - See [picking the right version](https://docs.silverstripe.org/en/contributing/code/#picking-the-right-version)
    - Small amounts of additional linting are usually okay, but if it makes it hard to concentrate on the relevant changes, ask for the unrelated changes to be reverted, and submitted as a separate PR.
